app/templates/main/volunteerProfile.html

```diff
@@ -15,7 +15,16 @@
 <div>
     <h1 align="center">{{volunteer.firstName}} {{volunteer.lastName}}</h1>
     <h6 align="center">{{volunteer.email}}</h6>
-    <h6 align="center">{{volunteer.phoneNumber}}</h6>
+    <div align="center">
+      <tr>
+        <td>
+          <input id="newinput" type="text" value="{{volunteer.phoneNumber}}">
+        </td>
+        <td>
+          <button class=" btn-success bi-pincel h-2" id="updatePhone" data-username="{{volunteer.username}}">Update</button>
+        </td>
+    </tr>
```

Comment:
Figure out how to do layout without tables. Especially when you aren't even including a `<table>` tag